test/Microsoft.AspNetCore.Identity.Test/SecurityStampValidatorTest.cs

```diff
@@ -86,7 +86,7 @@ public class SecurityStampTest
             contextAccessor.Setup(a => a.HttpContext).Returns(httpContext.Object);
             var signInManager = new Mock<SignInManager<TestUser>>(userManager.Object,
                 contextAccessor.Object, claimsManager.Object, options.Object, null);
-            signInManager.Setup(s => s.ValidateSecurityStampAsync(It.IsAny<ClaimsPrincipal>())).ReturnsAsync(null).Verifiable();
+            signInManager.Setup(s => s.ValidateSecurityStampAsync(It.IsAny<ClaimsPrincipal>())).ReturnsAsync(default(TestUser)).Verifiable();
```

Comment:
FYI Moq 4.7 added a new overload. This was required to disambiguate between two candidate overloads.